documentation/functionalDoc/functionalDoc.tex

```diff
@@ -58,31 +58,39 @@ \subsection{Generators}
+\scriptsize{\textbf{{Voltage regulation}}} & \scriptsize{\textbf{{InfiniteReactiveLimits}}} & \scriptsize{\textbf{{Rectangular diagram}}} & \scriptsize{\textbf{{Model}}}\\
 \midrule
 \rowcolor{white}
- \small{None}  & \small{-} & \footnotesize{fixed PQ generator} \\
+ \scriptsize{None}  & \scriptsize{-} & \scriptsize{-} & \scriptsize{fixed PQ generator} \\
+\rowcolor{gray!10}
+ \scriptsize{At the connection bus} & \scriptsize{false} & \scriptsize{false} & \scriptsize{GeneratorPVDiagramPQSignalN} \\
+\rowcolor{white}
+ \scriptsize{At the connection bus} & \scriptsize{false} & \scriptsize{true} & \scriptsize{GeneratorPVSignalN} \\
 \rowcolor{gray!10}
- \small{At the connection bus} & \small{false} & \footnotesize{GeneratorPVDiagramPQSignalN} \\
+ \scriptsize{At the connection bus} & \scriptsize{true} & \scriptsize{-} & \scriptsize{GeneratorPVSignalN} \\
 \rowcolor{white}
- \small{At the connection bus} & \small{true} & \footnotesize{GeneratorPVSignalN} \\
+ \scriptsize{At a distant bus} & \scriptsize{false} & \scriptsize{false} & \scriptsize{GeneratorPVRemoteDiagramPQSignalN} \\
 \rowcolor{gray!10}
- \small{At a distant bus} & \small{false} & \footnotesize{GeneratorPVRemoteDiagramPQSignalN} \\
+ \scriptsize{At a distant bus} & \scriptsize{false} & \scriptsize{true} & \scriptsize{GeneratorPVRemoteSignalN} \\
 \rowcolor{white}
- \small{At a distant bus} & \small{true} & \footnotesize{GeneratorPVRemoteSignalN} \\
+ \scriptsize{At a distant bus} & \scriptsize{true} & \scriptsize{-} & \scriptsize{GeneratorPVRemoteSignalN} \\
 \rowcolor{gray!10}
-\small{Another generator}&  &  \\
+\scriptsize{Another generator}&  &  & \scriptsize{GeneratorPQPropDiagramPQSignalN}\\
 \rowcolor{gray!10}
-\small{regulates the same node} & \multirow{-2}{*}{\small{false}} & \multirow{-2}{*}{\footnotesize{GeneratorPQPropDiagramPQSignalN}} \\
+\scriptsize{regulates the same node} & \multirow{-2}{*}{\scriptsize{false}} & \multirow{-2}{*}{\scriptsize{false}} & \scriptsize{+ VRRemote model} \\
 \rowcolor{white}
-\small{Another generator}&  & \footnotesize{GeneratorPQPropSignalN}  \\
+\scriptsize{Another generator}&  &  & \scriptsize{GeneratorPQPropSignalN}\\
 \rowcolor{white}
-\small{regulates the same node} & \multirow{-2}{*}{\small{true}} & \footnotesize{+ VRRemote model} \\
+\scriptsize{regulates the same node} & \multirow{-2}{*}{\scriptsize{false}} & \multirow{-2}{*}{\scriptsize{true}} & \scriptsize{+ VRRemote model} \\
+\rowcolor{gray!10}
+\scriptsize{Another generator}&  & & \scriptsize{GeneratorPQPropSignalN}  \\
+\rowcolor{gray!10}
+\scriptsize{regulates the same node} & \multirow{-2}{*}{\scriptsize{true}} & \multirow{-2}{*}{\scriptsize{-}} & \scriptsize{+ VRRemote model} \\
 \bottomrule
 \end{tabular}
-\caption{Generators modelling}
+\caption{Generators modelling. When a generator has a rectangular diagram, it means all the values of $Q_{min}$ are equal and all the values of $Q_{max}$ are equal}
```

Comment:
Maybe better to put this above the table?

Reply:
yes, I put it above and more explicitly, thx